spec/cypress/support/e2e.js

```diff
@@ -0,0 +1,2 @@
+import './commands'
+import './on-rails'
```

Comment:
Why didn't we need these packages beforehand? What's their use? Where is this new file `e2e.js` imported/used?

Reply:
It's automatically used by cypress per their new docs. They were imported differently before.